### PR TITLE
Add backup UUID for backup vm

### PR DESCRIPTION
**Pull request by shenitzky**

### CODE REVIEW

**Comment by shenitzky** [comment]
@mnecas @mkemel @pbar@redhat.com

**Comment by mnecas** [comment]
@barpavel

**Comment by mnecas** [comment]
ci test please

**Review by mnecas — APPROVED** [review]
LGTM

File: examples/backup_vm.py [hunk 1]
```diff
@@ -314,6 +264,34 @@ def add_download_args(parser):
...
+        "--disk-uuid",
+        action="append",
+        help="Disk UUID to backup. May be used multiple times to backup "
+             "multiple disks. If not specified, backup all VM disks.")
+
+    parser.add_argument(
+        "--skip-download",
+        dest="download_backup",
+        action="store_false",
+        help="Download full backup disks should be skipped. If specified, VM backup will "
```
Comment (barpavel) [inline examples/backup_vm.py:286]: It can be either "full" or "incremental" ot

File: examples/backup_vm.py [hunk 2]
```diff
@@ -54,98 +54,47 @@ def main():
...
-    start_parser.add_argument(
-        "vm_uuid",
-        help="UUID of the VM to backup.")
-
-    start_parser.add_argument(
-        "--disk-uuid",
-        action="append",
-        help="Disk UUID to backup. May be used multiple times to backup "
-             "multiple disks. If not specified, backup all VM disks.")
+    add_start_backup_args(start_parser)
```
Comment (barpavel) [inline examples/backup_vm.py:90]: Didn't have an allowed "--skip-download" argument before.

**Pull request merged**